tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
Ok - because you need to treat all these types the same and pass them around - let's keep them as classes for now @junchaowu @abhinav